.ci/install_qemu.sh

```diff
@@ -11,12 +11,12 @@ cidir=$(dirname "$0")
 source "${cidir}/lib.sh"
 source /etc/os-release || source /usr/lib/os-release
 
-CURRENT_QEMU_BRANCH=$(get_version "assets.hypervisor.qemu-lite.branch")
-CURRENT_QEMU_COMMIT=$(get_version "assets.hypervisor.qemu-lite.commit")
-QEMU_REPO_URL=$(get_version "assets.hypervisor.qemu-lite.url")
+CURRENT_QEMU_BRANCH=$(get_version "assets.hypervisor.qemu.version")
```

Comment:
since we are getting the version, please rename this variable to `CURRENT_QEMU_VERSION`